lib/config/options/index.ts

```diff
@@ -1962,6 +1962,12 @@ const options: RenovateOptions[] = [
     type: 'integer',
     default: 25,
   },
+  {
+    name: 'commitHourlyLimit',
+    description: 'Rate limit commits to maximum x per hour. 0 means no limit.',
+    type: 'integer',
+    default: 10,
```

Comment:
```suggestion
    default: 0,
```

 Default behavior should not be changed